brownie/project/main.py

```diff
@@ -725,13 +726,12 @@ def _install_from_github(package_id: str) -> str:
     if install_path.exists():
         raise FileExistsError("Package is aleady installed")
 
-    headers: Dict = {}
     if os.getenv("GITHUB_TOKEN"):
         auth = b64encode(os.environ["GITHUB_TOKEN"].encode()).decode()
-        headers = {"Authorization": "Basic {}".format(auth)}
+        REQUEST_HEADERS.update({"Authorization": "Basic {}".format(auth)})
```

Comment:
This will mutate `REQUEST_HEADERS` and leak into the calls to etherscan.  Maybe better to start by copying `REQUEST_HEADERS` to a local variable and then modifying?

Reply:
👍 Good catch, I was not aware of this problem